WooCommerce/Classes/ViewRelated/Dashboard/Settings/Plugins/PluginListCell/PluginTableViewCell.swift

```diff
@@ -0,0 +1,23 @@
+import UIKit
+
+ class PluginTableViewCell: UITableViewCell {
+
+     @IBOutlet private var nameLabel: UILabel!
+     @IBOutlet private var descriptionLabel: UILabel!
+
+     override func awakeFromNib() {
+         super.awakeFromNib()
+         applyDefaultBackgroundStyle()
+
+         nameLabel.applyBodyStyle()
+         nameLabel.numberOfLines = 2
+
+         descriptionLabel.applySecondaryBodyStyle()
+         descriptionLabel.numberOfLines = 2
+     }
+
+     func update(viewModel: PluginListCellViewModel) {
+         nameLabel.text = viewModel.name
+         descriptionLabel.text = viewModel.description
+     }
+ }
```

Comment:
I actually did look at `HeadlineLabelTableViewCell` but not sure if updating it can require regression tests to all other screens that use it so I chose the safer solution. But I'll take a look again and try to make changes that don't require change to other parts of the app.